_includes/configuration/toolbar_sticky_offset.md

```diff
@@ -0,0 +1,21 @@
+## toolbar_sticky_offset
+
+{{site.requires_5_9v}}
+
+This option allows the toolbar to stick or dock at the specified offset from the top or bottom of the view depending on the [toolbar mode]({{site.baseurl}}/configure/editor-appearance/#toolbar_mode}}). This can be useful when a site has it's own sticky header navigation bar to ensure the toolbar sticks below.
```

Comment:
```suggestion
This option allows the toolbar to stick or dock at the specified offset from the top or bottom of the [viewport](https://developer.mozilla.org/en-US/docs/Glossary/Viewport) depending on the [toolbar location (`toolbar_location`)]({{site.baseurl}}/configure/editor-appearance/#toolbar_location}}). This can be useful when a site has a sticky header navigation bar, allowing the {{site.productname}} toolbar to "stick" below the site's navigation bar.
```